app/app_test.go

```diff
@@ -69,45 +88,51 @@ func (at *appTest) reset() {
 	require.True(at.t, resabci.IsOK(), resabci)
 }
 
+func getBalance(pk crypto.PubKey, state types.KVStore) (types.Coins, error) {
+	return getAddr(pk.Address(), state)
+}
+
+func getAddr(addr []byte, state types.KVStore) (types.Coins, error) {
+	actor := stack.SigPerm(addr)
+	acct, err := coin.NewAccountant("").GetAccount(state, actor)
```

Comment:
Okay - `acct` it is - opens up the new issue of maintaining consistency there are many instances of `acc` - sometimes as a part of a larger variable name such as `accIn`